### ISSUE TITLE
Use **kwargs for additional configuration

### ISSUE THREAD

**Issue by jornfranke** [ZuInnoTe/scrapy-contrib-bigexporters#2]
At the moment, we have for each of the format dedicated key/values in settings.py.
Additionally we could use **kwargs to read those values. This will allow us to better support the FEEDS settings where multiple exporter can be defined with different **kwargs.

Then, the following would be possible - example with parquet - works analogous with other formats.
````
items.parquet': {
        'format': 'parquet',
        'encoding': 'utf8',
        'store_empty': False,
        'item_export_kwargs': {
           'compression': 'GZIP'
        },
    }
```
Here the configuration is stored in item_export_kwargs
This means we would deprecate and later remove the analgous settings in settings.py

**Comment by jornfranke** [comment]
Implies also that the minimal Scrapy version should be 2.3.0

**Comment by jornfranke** [comment]
implemented in 0.2
